lms/static/js/student_profile/views/badge_list_container.js

```diff
@@ -0,0 +1,25 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'common/js/components/views/paginated_view',
+        'js/student_profile/views/badge_view', 'js/student_profile/views/badge_list_view',
+        'text!templates/student_profile/badge_list.underscore'],
+        function (gettext, $, _, PaginatedView, BadgeView, BadgeListView, BadgeListTemplate) {
```

Comment:
@andy-armstrong Will the tests at https://github.com/edx/edx-platform/pull/10861/files#diff-6e10e312368861428d70a50f73284b1cR71 Suffice for this?